src/Algebra/Module/Bundles.agda

```diff
@@ -333,3 +368,6 @@ record Module (commutativeRing : CommutativeRing r ℓr) m ℓm
   semimodule = record { isSemimodule = isSemimodule }
 
   open Semimodule semimodule public using (*ₗ-comm; *ᵣ-comm)
+
+  -ᴹ-involutive : Involutive _≈ᴹ_ -ᴹ_
```

Comment:
The bundle (i.e. `Semimodule`) should not contain any proofs that the structures (i.e. `IsSemimodule`) does not have. Therefore this should go in `Algebra.Module.Properties.Semimodule` (this file may not yet exist).

Reply:
You meant to say `Module`, right?
(This property is defined inside the `Module` record and relies on `-ᴹ_`.)

Reply:
So, I just noticed this in `Algebra.Module.Bundles.Semimodule`:

```agda
*ₗ-comm : L.Commutative _*ₗ_
*ₗ-comm x y m = begin
  x *ₗ y *ₗ m   ≈⟨ ≈ᴹ-sym (*ₗ-assoc x y m) ⟩
  (x * y) *ₗ m  ≈⟨ *ₗ-cong (*-comm _ _) ≈ᴹ-refl ⟩
  (y * x) *ₗ m  ≈⟨ *ₗ-assoc y x m ⟩
  y *ₗ x *ₗ m   ∎

*ᵣ-comm : R.Commutative _*ᵣ_
*ᵣ-comm m x y = begin
  m *ᵣ x *ᵣ y   ≈⟨ *ᵣ-assoc m x y ⟩
  m *ᵣ (x * y)  ≈⟨ *ᵣ-cong ≈ᴹ-refl (*-comm _ _) ⟩
  m *ᵣ (y * x)  ≈⟨ ≈ᴹ-sym (*ᵣ-assoc m y x) ⟩
  m *ᵣ y *ᵣ x   ∎
```

And I'm wondering why this exception was made.
Thanks!

Reply:
Here's what I ended up doing:

```
dbanas@DESKTOP-G84ND7C:~/prj/agda-stdlib
$ tree src/Algebra/Module/
src/Algebra/Module/
{snip}
├── Properties
│   ├── Semimodule.agda - Contains the old contents of `../Properties.agda`.
├── Properties.agda     - Provides `-ᴹ-involutive` and
                           imports (publicly) `Properties/Semimodule.agda.
{snip}
```

Let me know if you'd like further changes.

Reply:
@MatthewDaggitt ?